src/main.py

```diff
@@ -76,16 +69,8 @@ def is_yaml_file(f: str) -> bool:
     return f.endswith(".yml") or f.endswith(".yaml")
 
 
-async def handle_event(event: Dict, event_context, projects_ids: Optional[List[str]] = None, services: Optional[List[GCPService]] = None):
-    if isinstance(event_context, Dict):
-        # for k8s installation
-        context = LoggingContext(event_context.get("execution_id", None))
-    else:
-        context = LoggingContext(None)
-
-    if not services:
-        # load services for GCP Function
-        services = load_supported_services(context)
+async def query_metrics(execution_id: Optional[str], services: Optional[List[GCPService]] = None):
```

Comment:
I think handle_event is execution start point for cloud function? And probably we need such function for now